<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonFind">
        <property name="text">
         <string>Искать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeSelect">
        <item>
         <property name="text">
          <string>JSON</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CSV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Новый элемент</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MyTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editID"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editTitle"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Auditory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editAuditory"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Тип занятия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboType">
        <item>
         <property name="text">
          <string>lecture</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>practice</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>laboratory</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>exam</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonEnd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>mytextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonFind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>findAuditory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEnd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addToVector()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findAuditory()</slot>
  <slot>addToVector()</slot>
  <slot>ComboResult(QString)</slot>
 </slots>
</ui>
